tests/extension/khr_free_function_commands/free_function_commands.cpp

```diff
@@ -0,0 +1,657 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright (c) 2025 The Khronos Group Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+*******************************************************************************/
+
+#include "../../common/common.h"
+
+namespace free_function_commands::tests {
+
+#ifdef SYCL_KHR_FREE_FUNCTION_COMMANDS
+static void test_submit() {
+  sycl::queue q;
+  constexpr int val = 314;
+  int* buf = sycl::malloc_shared<int>(1, q);
```

Comment:
`malloc_shared` may throw an exception if `usm_shared_allocations` aspect is not supported. I don't think that it should be considered as a test failure and the test should be skipped in that case

Reply:
Changed the code to use sycl::buffers where possible and added an if-check for malloc_shared - [67c3b83](https://github.com/KhronosGroup/SYCL-CTS/pull/1027/commits/67c3b83a602da168ebf89b38ab2e8aca08a6a0ab)